packages/lodestar/src/network/nodejs/bundle.ts

```diff
@@ -8,12 +8,13 @@ import Mplex from "libp2p-mplex";
 import {NOISE} from "libp2p-noise";
 import Bootstrap from "libp2p-bootstrap";
 import MDNS from "libp2p-mdns";
-import PeerInfo from "peer-info";
+import PeerId from "peer-id";
 import {ENRInput, Discv5Discovery} from "@chainsafe/discv5";
 
 
 export interface ILibp2pOptions {
-  peerInfo: PeerInfo;
+  peerId: PeerId;
+  listenAddrs: string[];
```

Comment:
Might be a good time to support all three, {listen, announce, noAnnounce}, even if its not yet supported downstream in the cli. Eg: for now we can do: `NodejsNode({addresses: {listen: multiaddrs}})`, later adding options to configure announce and noAnnounce.